### DISCUSSION TITLE
Output friendly name in attributes

### DISCUSSION THREAD

**Discussion by wzrdtales** [node-saml/node-saml#324] [General]
Currently, it seems like only the attribute name is respected, and not possibility of outputting the friendly name exists yet. Or did I miss this somehow?

**Comment by srd90** [comment]
FYI there is an open issue with some discussion at `@node-saml/passport-saml` side. Issue was created before split to framework agnostic core saml (`@node-saml/node-saml`) and passportjs & expressjs specific (`@node-saml/passport-saml` which uses internally `@node-saml/node-saml`) happened at versions 4.0.0

* https://github.com/node-saml/passport-saml/issues/282
   * https://github.com/node-saml/passport-saml/pull/389
